modules/admin_manual/pages/configuration/server/occ_commands/core_commands/_previews_commands.adoc

```diff
@@ -0,0 +1,58 @@
+= Previews Commands
+
+A set of commands to remove unreferenced preview images:
+
+[source,plaintext]
+----
+previews
+ previews:cleanup            Remove unreferenced previews
+----
+
+== Cleanup
+
+Removing unreferenced previews can be necessary, like when the image has been deleted or the ID of the moint point changes which happens when renaming it. Previews will be regenerated when reaccessing the image, but unreferenced previews are orphanes. To circumvent orphans, a occ command has been added to manually remove those unreferenced previews as they are eating up storage unneccessarily. In addition, a backgroundjob has been created which automatically removes them on a regulary basis. For more info see the xref:background-job[Background Job] section and the example listing xref:configuration/server/occ_command.adoc#list-queued-backgroundjobs[OCA\Files\BackgroundJob\PreviewCleanupJob].
```

Comment:
```suggestion
Removing not referenced previews can be necessary, e.g., when the image has been deleted or the ID of the mountpoint changes which happens when renaming it. Previews will be generated anew when accessing the image again, but not referenced previews are orphans. To avoid orphans, an occ command has been added to manually remove those not referenced previews as they are eating up storage space unnecessarily. In addition, a background job has been created which automatically removes them on a regular basis. For more info see the xref:background-job[Background Job] section and the example listing xref:configuration/server/occ_command.adoc#list-queued-backgroundjobs[OCA\Files\BackgroundJob\PreviewCleanupJob].
```